static/css/perdiem.css

```diff
@@ -299,3 +299,22 @@ ul.tabs li {
     margin: 0 auto;
     padding: 20px;
 }
+
```

Comment:
![screen shot 2016-05-26 at 10 32 47 pm](https://cloud.githubusercontent.com/assets/17802940/15598729/ec61cb80-2391-11e6-9a76-4dd947ca7514.png)

Got an error when adding that code to the page